src/IK/src/CoMTask.cpp

```diff
@@ -235,3 +245,14 @@ bool CoMTask::isValid() const
 {
     return m_isValid;
 }
+
+bool CoMTask::setFeedback(Eigen::Ref<const Eigen::Vector3d> feedback)
+{
+    bool ok = true;
+    if (m_useExogenousFeedback)
+    {
+        ok = ok && m_R3Controller.setState(feedback);
+    }
+
+    return ok;
```

Comment:
Should it return false if ``m_useExogenousFeedback`` is false?

Reply:
For the SE3Task we return true if the flag is false. Still it makes sense to return false

Reply:
With a debug print

Reply:
As discussed F2F, it may be worth returning ``true`` anyhow so that we can keep the call and just use the flag to turn the feature on and off